templates/docs/examples/patterns/split-list.html

```diff
@@ -0,0 +1,522 @@
+      </div>
+
+      <div>
+        <div>
+          <h5>
+            Rich portfolio of products
+          </h5>
+        </div>
+
+        <div>
+          <p>
+            Resell the full range of Canonical’s portfolio of private and public
+            cloud products: Openstack, Kubernetes, IoT, support, security and
+            compliance for Ubuntu.
+          </p>
+        </div>
+      </div>
+
+      <div>
+        <div>
+          <hr>
+        </div>
+      </div>
+
+      <div>
+        <div>
+          <h5>
+            Sales enablement
+          </h5>
+        </div>
+
+        <div>
+          <p>
+            Canonical can help train your field sales and presales teams and
+            provide you with sales collateral and permission to use the Canonical
+            and Ubuntu trademarks in your own materials.
+          </p>
+        </div>
+      </div>
+    </div>
+  </div>
+</div>
+
+<h6>full width, full width, 25-25-75 (Jinja macro)</h6>
+
+{{ split_list(variant=1,
+   title='H2 - this can take up to two lines, but best to stick to one line.',
+   list_items=[{
+     'title': 'Rich portfolio of products',
+     'description': 'Resell the full range of Canonical’s portfolio of private and public cloud products: Openstack, Kubernetes, IoT, support, security and compliance for Ubuntu.'
+   }, {
+     'title': 'Sales enablement',
+     'description': 'Canonical can help train your field sales and presales teams and provide you with sales collateral and permission to use the Canonical and Ubuntu trademarks in your own materials.'
+   }]) }}
+
+<h6>full width, 50-50, 25-25-75 (Jinja macro)</h6>
+
+{{ split_list(variant=2,
+   title='H2 - this can take up to two lines, but best to stick to one line.',
+   list_items=[{
+     'title': 'Rich portfolio of products',
+     'description': 'Resell the full range of Canonical’s portfolio of private and public cloud products: Openstack, Kubernetes, IoT, support, security and compliance for Ubuntu.'
+   }, {
+     'title': 'Sales enablement',
+     'description': 'Canonical can help train your field sales and presales teams and provide you with sales collateral and permission to use the Canonical and Ubuntu trademarks in your own materials.'
+   }]) }}
+
+<h6>full width, 50-50, 50-50 (Jinja macro)</h6>
+
+{{ split_list(variant=3,
+   title='H2 - this can take up to two lines, but best to stick to one line.',
+   list_items=[{
+     'title': 'Rich portfolio of products',
+     'description': 'Resell the full range of Canonical’s portfolio of private and public cloud products: Openstack, Kubernetes, IoT, support, security and compliance for Ubuntu.'
+   }, {
+     'title': 'Sales enablement',
+     'description': 'Canonical can help train your field sales and presales teams and provide you with sales collateral and permission to use the Canonical and Ubuntu trademarks in your own materials.'
+   }]) }}
+
+<h6>full width, full width, 50-50 (Jinja macro)</h6>
+
+{{ split_list(variant=4,
```

Comment:
Variant would need to be something more user friendly.

If there are two separate options, maybe there should be 2 props:
layout-large="50/50" | "100", etc
layout-medium="50/50" | "100", etc

with one of them being a default, so in most common case you don't have to define it at all?